swagger.yaml

```diff
@@ -3567,6 +3567,12 @@ definitions:
           $ref: '#/definitions/interface'
       disks:
         type: array
+        description: "Disks that are candidates for installation, filtered"
+        items:
+          $ref: '#/definitions/disk'
+      unfiltered_disks:
+        type: array
+        description: "All disks found on the system, unfiltered"
```

Comment:
Instead of creating another list, why not add a boolean to the "disk" definition called "eligible"?  This way the UI can also choose to show the extra disks more easily and there is no need to compare two lists to see what was filtered out.

Reply:
This has been discussed in the Jira issue - in short, adding a new field to the existing list will require changes in existing code and future code to make sure that it considers this field when working on this list

Reply:
Thanks, replied in the ticket